compiler/passes/insertWideReferences.cpp

```diff
@@ -1878,15 +1878,21 @@ static void insertWideTemp(Type* type, SymExpr* src) {
   Type* srcType = src->symbol()->type;
 
   bool needsAddrOf = false;
-  bool needsDeref = false;
+  bool needsDeref  = false;
+  bool widenVal    = false;
 
   if (type->isRefOrWideRef() &&
-      !src->isRefOrWideRef())
+      !src->isRefOrWideRef()) {
     needsAddrOf = true;
-
-  if (src->isRefOrWideRef() &&
-      !type->isRefOrWideRef())
+  }
```

Comment:
Tiny niggle that you can ignore.  To my eye the brace convention on this else if
is inconsistent with very next else-if.  Brace style is somewhat personal but I
think it's good to be consistent.